src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -120,6 +170,265 @@ public void recordTagValue(String time)
       }
    }
 
+   /**
+    * exportBlockDescriptor
+    * Send an EBD command with a start and end time and a tag name.
+    * Start time is stored in an external file, end time is current time.
+    */
+   public void exportBlockDescriptor()

```

Comment:
Should this be names something else? It seems to be a very specific EBD request and not a generic method for all EBDs.